### ISSUE TITLE
Concern over deprecated project dependencies

### ISSUE THREAD

**Issue by cwdott** [vuejs-templates/webpack#1506]
Just installing a new project with `vue init webpack <project>` and I'm getting all of the following warnings:

`npm WARN deprecated extract-text-webpack-plugin@3.0.2: Deprecated. Please use https://github.com/webpack-contrib/mini-css-extract-plugin`
`npm WARN deprecated browserslist@2.11.3: Browserslist 2 could fail on reading Browserslist >3.0 config used in other tools.`
`npm WARN deprecated bfj-node4@5.3.1: Switch to the `bfj` package for fixes and new features!`
`npm WARN deprecated flatten@1.0.2: I wrote this module a very long time ago; you should use something else.`
`npm WARN deprecated browserslist@1.7.7: Browserslist 2 could fail on reading Browserslist >3.0 config used in other tools.`
`npm WARN deprecated circular-json@0.3.3: CircularJSON is in maintenance only, flatted is its successor.`

Should I concerned about these dependencies being deprecated?

**Comment by ararename** [comment]
I would love to learn about that, too. 
Also other templates i found are outdated. 
Is there another way of calling webpack to ptoduce something usable?

**Comment by ararename** [comment]
Right, i do have different deptecation warnings and this if i try to reproduce: 

found 92 vulnerabilities (69 low, 9 moderate, 13 high, 1 critical)
  run `npm audit fix` to fix them, or `npm audit` for details

This is not the best way to get going with a starter template. 

$ npm audit fix
fixed 8 of 92 vulnerabilities in 32518 scanned packages
  1 vulnerability required manual review and could not be updated
  8 package updates for 83 vulnerabilities involved breaking changes

What is the advice for a beginner on how to deal with this?
